.expeditor/verify_public.pipeline.yml

```diff
@@ -574,8 +574,9 @@ steps:
 
 - label: "Spellcheck"
   commands:
+    - ruby -rjson -e "JSON.parse(File.read('cspell.json'))" 2>/dev/null || (echo "Failed to parse config file 'cspell.json', skipping spellcheck" && exit 1)
```

Comment:
I tried to use node here, but because cspell uses a non-standard form of json that allows comments, node's json parser blew up but ruby's worked.